src/frontend/screens/Library/components/GameCard/index.tsx

```diff
@@ -338,6 +338,11 @@ const GameCard = ({
       onclick: () => favouriteGames.add(appName, title),
       show: !isFavouriteGame
     },
+    {
+      label: t('submenu.manageCategories', 'Manage Categories'),
```

Comment:
```suggestion
      label: t('submenu.categories', 'Categories'),
```

I think it's still clear and makes it easier for translators (in some languages maybe it's long to say "Manage Categories" and then the card menu might look weird)